tomcat-conf/Catalina/localhost/prweb.xml

```diff
@@ -32,5 +32,13 @@
   <Environment name="prconfig/session/ha/enabled" value="true" type="java.lang.String"/>
   <Environment name="prconfig/dsm/services/stream/pyBaseLogPath" value="/opt/pega/kafkadata" type="java.lang.String" />
   <Environment name="prconfig/cloud/isNodeAgnosticAdminStudio" value="true" type="java.lang.String"/>
+
+  {{ if isTrue .Env.CLIENT_MODE }}
```

Comment:
As a parameter, `CLIENT_MODE` sounds like this node will be accepting clients IMO.  Can we go with something more descriptive that is user facing? We had previously called this the cluster service, correct?Maybe something like `CLUSTER_SERVICE_URL` or `DATA_GRID_URL`. We could infer we should enable this mode if a URL is provided.  Thoughts?

Reply:
Ok, instead of checking CLIENT_MODE as true , I could infer this based on whether URL is provided or not. It will then look something like this:-
{{ if isTrue .Env.URL}}